README.md

```diff
@@ -28,7 +28,7 @@ To work on the project:
 
 If you’ve tried AltspaceVR on Android, you'll have noticed that we have a number of scrollable views that show preview images of the spaces and events you can visit. In this app, you'll apply similar techniques in order to let users search and browse images and save the ones they like.
 
-The example project should provide a good foundation for the app. The places in the code you need to implement are commented `YOUR CODE HERE`. Also, there is a small example unit test in place, feel free to add additional tests as needed. For your reference, we've also included a sample completed `ImageSearcher.APK` in the Project directory that demonstrates what you should have working after finishing Part 1.
+The example project should provide a good foundation for the app. The places in the code you need to implement are commented `YOUR CODE HERE`. Also, there is a small example unit test in place, feel free to add additional tests as needed.
```

Comment:
The orig APKs were taken out since they were not obfuscated. The one added now is, so we should keep the reference as is.